lib/dialects/sqlite/query-interface.js

```diff
@@ -210,7 +210,7 @@ class SQLiteQueryInterface extends QueryInterface {
       }
       for (const index of indexes) {
         for (const field of index.fields) {
-          if (index.unique !== undefined) {
+          if (index.unique !== undefined && index.fields.length === 1) {
```

Comment:
Updated this condition to only set a column to unique if the index contains 1 column. This allows existing behavior to stay in place for single column indexes.

Multi-column indexes are NOT persisted when tables are rebuilt. I'm not sure how this is preferred in Sequelize code base.